x-pack/legacy/plugins/siem/public/components/flyout/index.test.tsx

```diff
@@ -57,41 +52,13 @@ describe('Flyout', () => {
       ).toContain('Timeline');
     });
 
-    test('it renders the title field when its state is set to flyout is true', () => {
-      const stateShowIsTrue = set('timeline.timelineById.test.show', true, state);
-      const storeShowIsTrue = createStore(stateShowIsTrue, apolloClientObservable);
-
-      const wrapper = mount(
-        <TestProviders store={storeShowIsTrue}>
-          <Flyout
-            flyoutHeight={testFlyoutHeight}
-            headerHeight={flyoutHeaderHeight}
-            timelineId="test"
-            usersViewing={usersViewing}
-          />
-        </TestProviders>
-      );
-
-      expect(
-        wrapper
-          .find('[data-test-subj="timeline-title"]')
-          .first()
-          .props().placeholder
-      ).toContain('Untitled Timeline');
```

Comment:
Is there an equivalent version of this test?

Reply:
https://github.com/elastic/kibana/pull/59587/files#diff-0783c66236e96928476bfba4b76d1fd3R104